changelog/18228.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+hybrid-management: Add OSS support for hybrid management
```

Comment:
`hybrid-management` is neat! 
Are you going to change `hcplink` references in the code as well?

Reply:
Possibly, but maybe in a separate PR. We might want to wait until we align on something.